content/docs/operating/configuration.md

```diff
@@ -202,6 +206,32 @@ services:
 [ tag_separator: <string> | default = , ]
 ```
 
+### Zookeeper Serverset SD configurations `<serverset_sd_config>`
+
+Serverset SD configuration SD configurations allow retrieving scrape targets
+from [Serversets]
+(https://github.com/twitter/finagle/tree/master/finagle-serversets) which are
+stored in [Zookeeper](https://zookeeper.apache.org/). Serversets are commonly
+used by Finagle and Aurora.
+
+The following meta labels are available on targets during relabeling:
+
+* `__meta_serverset_path`: the full path to the serverset member node in Zookeeper
+* `__meta_serverset_endpoint_host`: the host of the default endpoint
+* `__meta_serverset_endpoint_port`: the port of the default endpoint
+* `__meta_serverset_endpoint_host_<endpoint>`: the host of the given endpoint
+* `__meta_serverset_endpoint_port_<endpoint>`: the port of the given endpoint
+* `__meta_serverset_status`: the status of the member
+
+```
+# The Zookeeper servers.
+servers: [ <host:post> ]
+# Path can point to a single serverset, or the root of a tree of serversets.
+path: <string>
```

Comment:
The correct field is `paths` as a list, right?
For servers `, ...` for list indication would be good. `post` -> `port`, though we just went with `<host>` as this is consistent with Go.

```
servers: [ <host>, ... ]
paths:
  [ - <string> ]
```